AdobeStockImage/Model/SaveMediaGalleryAsset.php

```diff
@@ -61,11 +62,13 @@ public function execute(Document $document, string $destinationPath): void
     {
         try {
             $fileSize = $this->calculateFileSize($destinationPath);
+            $hashedImageContent = $this->hashImageContent($destinationPath);
```

Comment:
We can use the `hashImageContent` directly when assigning value to the array key. Please, remove this line.